_locales/ru/messages.json

```diff
@@ -826,6 +826,24 @@
   "proMessageInfoFeatures": "Это сообщение использовало следующие функции Session Pro:",
   "proSendMore": "Отправить еще с",
   "profile": "Профиль",
+  "proxyAuthPassword": "Пароль (опционально)",
+  "proxyAuthUsername": "Имя пользователя (опционально)",
+  "proxyDescription": "Настройка SOCKS5 прокси для маршрутизации всего трафика приложения, включая onion-запросы, через прокси-сервер. Это может помочь обойти сетевые ограничения и глубокую проверку пакетов (DPI).",
+  "proxyEnabled": "Включить прокси",
+  "proxyHost": "Прокси-сервер",
+  "proxyHostPlaceholder": "например 127.0.0.1 или proxy.example.com",
+  "proxyPort": "Порт",
+  "proxyPortPlaceholder": "например 1080",
```

Comment:
those  will need to be added to crowdin first, as what comes from crowdin will overwrite those .json files.
We need your english strings on crowdin first though, so I'll let you know once it's ready to be translated.